### ISSUE TITLE
GPS is always on

### ISSUE THREAD

**Issue by ChaosCoder** [TICESoftware/tice-android#29]
The GPS access is on whenever the app is active according to this issue: https://github.com/TICESoftware/TICE/issues/7

We should make sure to only activate GPS when TICE needs it (we share our location).

**Comment by victor-sm** [comment]
Yes please. If I battery optimization is turned on, the app doesn't work correctly. If it's off, I loose 20% to 30% of my battery. This makes the app in its current state not usable.

I don't know how the protocol works, but a request should be sent to the app to wake up, when someone wants to see a position, otherwise the app should be dormant.
